File: 2022/March/Day 18/twoSum.cpp
// Source : https://leetcode.com/problems/two-sum/
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {

        vector<int> out;
        for(int i=0;i<size(nums);i++)
            for(int j=i+1;j<size(nums);j++)
            {
                if(i==j)
                    continue;
                if(nums[i]+nums[j]==target)
                {
                   out = {i,j};
                   return out;
                }
            }
        return out;
    }
};

File: 2024/Aug/Day 25/twosum.rs
// Source : https://leetcode.com/problems/two-sum/
impl Solution {
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
        let mut result1:Vec<i32> = Vec::new();
        for  x in (0..nums.len()){
            for y in (x+1..nums.len()){
                    if nums[x] + nums[y] == target{
                    result1.push(x as i32);
                    result1.push(y as i32);
                    return result1;
                    }

                } ;
            }
        
        return result1;
    }
}